### ISSUE TITLE
Make the output pretty

### ISSUE THREAD

**Issue by lu-zero** [rust-av/av-metrics#27]
You may use [console](https://crates.io/crates/console) to colorize the output and print it in tabular form (e.g. what [bat](https://crates.io/crates/console) does)

**Comment by monst12** [comment]
Hey, I would like working on this.
Is there anything I should consider before I start?

**Comment by shssoichiro** [comment]
Thanks! I don't think there are any special considerations. This work will be in the `av_metrics_tool` module of this project, if you're looking for where to get started. (`av_metrics` is the library portion and `av_metrics_tool` is the CLI interface.)

**Comment by monst12** [comment]
Hey, finally I made some examples of what I think could look somehow good.

In the first one I kept a similiar layout and tried to clarify used techniques and systems.
![Lines](https://user-images.githubusercontent.com/7419293/67523193-6f168500-f6ae-11e9-95b6-3777287d4c9c.PNG)

I really liked the way the header looked and just changed the output format into a table (for this one I additionally used [prettytable](https://github.com/phsym/prettytable-rs), which also can export the data to a csv file):
![Tables](https://user-images.githubusercontent.com/7419293/67523490-0bd92280-f6af-11e9-9d1b-987a58a04c85.PNG)

Maybe the table has a little bit too much text and is too colorful. It's also harder to get the CIEDE2000 metrics into the same table without introducing too much empty space.
For both the result text may could be better written as: `Results for the comparison of *** to ***`.

I would be happy about any feedback, especially concerning the colors and formatting.
